Pipfile

```diff
@@ -10,4 +10,3 @@ junit-xml = "*"
 [dev-packages]
 
 [requires]
-python_version = "2.7"
```

Comment:
Should this not be "2.7,3.7", to reflect what's being used in the Travis config?

Reply:
I believe that leaving it out is in alignment with the docs...
See https://pipenv-fork.readthedocs.io/en/latest/basics.html#specifying-versions-of-python

Reply:
Ah, sure :)